helm/ph-ee-engine/vouchers/templates/deployment.yaml

```diff
@@ -0,0 +1,87 @@
+{{- if .Values.deployment.annotations }}
+{{ toYaml .Values.deployment.annotations | indent 8 }}
+{{- end }}
+    spec:
+      containers:
+        - name: ph-ee-vouchers
+          image: "{{ .Values.image }}:{{ .Values.imageTag }}"
+          imagePullPolicy: "{{ .Values.imagePullPolicy }}"
+          ports:
+            - containerPort: 5000
+          resources:
+            limits:
+              memory: "{{ .Values.limits.memory }}"
+              cpu: "{{ .Values.limits.cpu }}"
+            requests:
+              memory: "{{ .Values.requests.memory }}"
+              cpu: "{{ .Values.requests.cpu }}"
+          env:
+            - name: SERVER_PORT
+              value: "{{ .Values.server.port }}"
+            - name: SPRING_DATASOURCE_URL
+              value: "{{ .Values.spring.datasource.url }}"
+            - name: SPRING_DATASOURCE_USERNAME
+              value: "{{ .Values.spring.datasource.username }}"
+            - name: SPRING_DATASOURCE_PASSWORD
+              value: "{{ .Values.spring.datasource.password }}"
+            - name: SPRING_DATASOURCE_DRIVER_CLASS_NAME
+              value: "{{ .Values.spring.datasource.driver-class-name }}"
+            - name: SPRING_LIQUIBASE_ENABLED
+              value: "{{ .Values.spring.liquibase.enabled }}"
+            - name: SPRING_LIQUIBASE_DEFAULT_SCHEMA
+              value: "{{ .Values.spring.liquibase.default-schema }}"
+            - name: SPRING_LIQUIBASE_CHANGE_LOG
+              value: "{{ .Values.spring.liquibase.change-log }}"
+            - name: SPRING_LIQUIBASE_CLEAR_CHECKSUMS
+              value: "{{ .Values.spring.liquibase.clear-checksums }}"
+            - name: LOGGING_LEVEL_ROOT
+              value: "{{ .Values.logging.level.ROOT }}"
+            - name: ASYNC_CORE_POOL_SIZE
+              value: "{{ .Values.async.core_pool_size }}"
+            - name: ASYNC_MAX_POOL_SIZE
+              value: "{{ .Values.async.max_pool_size }}"
+            - name: ASYNC_QUEUE_CAPACITY
+              value: "{{ .Values.async.queue_capacity }}"
+            - name: EXPIRY_TIME
+              value: "{{ .Values.expiry_time }}"
+            - name: SALTING_ENABLED
+              value: "{{ .Values.salting.enabled }}"
+            - name: IDENTITY_ACCOUNT_MAPPER_HOSTNAME
+              value: "{{ .Values.identity-account-mapper.hostname }}"
+            - name: ZEEBE_CLIENT_MAX_EXECUTION_THREADS
+              value: "{{ .Values.zeebe.client.max-execution-threads }}"
+            - name: ZEEBE_CLIENT_EVENLY_ALLOCATED_MAX_JOBS
+              value: "{{ .Values.zeebe.client.evenly-allocated-max-jobs }}"
+            - name: ZEEBE_BROKER_CONTACTPOINT
+              value: "{{ .Values.zeebe.broker.contactpoint }}"
+{{- if .Values.extraEnvs | default .Values.deployment.extraEnvs }}
+{{ toYaml ( .Values.extraEnvs | default .Values.deployment.extraEnvs ) | indent 10 }}
+{{- end }}        
+          envFrom: {{ toYaml ( .Values.envFrom | default .Values.deployment.envFrom ) | nindent 12 }}
```

Comment:
can you try adding Liveness and readiness check

Reply:
When adding Liveness and readiness check pod was not coming up